diagnostic.md

```diff
@@ -35,5 +43,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each contact.number as |number|}}
+      {{my-contact/my-phone number=number}}
+    {{/each}}
```

Comment:
you don't need `my-contact/` in the subcomponent.  So it should look something like this:
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```